UI/window-basic-main-dropfiles.cpp

```diff
@@ -72,6 +72,20 @@ void OBSBasic::AddDropURL(const char *url, QString &name, obs_data_t *settings,
 		cx = query.queryItemValue("layer-width").toInt();
 	if (query.hasQueryItem("layer-height"))
 		cy = query.queryItemValue("layer-height").toInt();
+	if (query.hasQueryItem("layer-css")) {
+		// QUrl::FullyDecoded does NOT properly decode a
+		// application/x-www-form-urlencoded space represented as '+'
```

Comment:
I have a feeling this is because JavaScript's `decodeURIComponent()` doesn't either, so it's likely by design. As long as you throw your CSS into `encodeURIComponent()` before sending it to OBS, `QUrl::FullyEncoded` should be plenty.

Reply:
That's what I expected as well, but during testing i consistently kept having spaces remain as +'s in the decoded string.

I'll gladly be corrected, though!

For reference, https://github.com/Marenthyu/PointBar/blob/b276e53d9499ca0542622593722e52239a153024/js/form.js#L126 is what i used for testing, which _should_ be encoding the parameter properly, shouldn't it? Calling ``encodeURIComponent`` on the value explicitly only encodes it twice, which is definitely not what I'd want to achieve.